shared/tracker/remote-proxy.desktop.js

```diff
@@ -35,16 +35,16 @@ const trackerMapStateToProps = (state: TypedState, {name}) => {
     description: showTeam && showTeam.description,
     following: state.config.following,
     loggedIn: state.config.loggedIn,
-    teamJoinError: state.entities.teams.teamJoinError,
-    teamJoinSuccess: state.entities.teams.teamJoinSuccess,
+    teamJoinError: state.teams.teamJoinError,
+    teamJoinSuccess: state.teams.teamJoinSuccess,
     memberCount: showTeam && showTeam.numMembers,
     openTeam: showTeam && showTeam.open,
     publicAdmins,
     publicAdminsOthers,
     showTeam: showTeam || '',
     teamname,
-    youAreInTeam: !!state.entities.getIn(['teams', 'teamnames', teamname], false),
-    youHaveRequestedAccess: !!state.entities.getIn(['teams', 'teamAccessRequestsPending', teamname], false),
+    youAreInTeam: !!state.teams.getIn(['teamnames', teamname], false),
```

Comment:
I'm not sure these are getting piped through correctly, I see `Request to join` etc. for teams I'm already in on the tracker popup, but it works correctly on the profile page.

Reply:
Hmm, I can't reproduce -- seems to work for me with your tracker popup and `keybasefriends` (which I'm also a member of). Can you try again, and maybe we can look through it together tomorrow?

Reply:
Ah it does work for me now, nevermind!